projects/vdk-core/src/vdk/internal/builtin_plugins/ingestion/ingester_configuration_plugin.py

```diff
@@ -29,6 +29,14 @@ def vdk_configure(self, config_builder: ConfigurationBuilder) -> None:
             default_value=None,
             description="Default Ingestion target to be used.",
         )
+        config_builder.add(
+            key="INGEST_PLUGIN_EXECUTION_SEQUENCE",
+            default_value=None,
+            description="""A string of coma-separated ingestion plugin method names,
```

Comment:
Maybe some short example to clarify what you mean here ...

Reply:
Added.